_posts/2016-04-20-squabble-comments.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "How we added comments to our Jekyll site"
+date: 2016-04-20
+author: Kosta Harlan
+tags: jekyll
+summary: |
+  A peek under the hood at our website's commenting app, [Squabble](http://github.com/savaslabs/squabble).
+---
+[Avid readers of our blog](/2015/04/01/building-our-site.html) know that it's built using the amazing and lovely open-source static site generator known as [Jekyll](http://jekyllrb.com/).
+
+Jekyll is a great solution for our site for a site like ours because we get an always-on[^1], super fast, database-free site that's also capable of many of the things you'd typically associate with a database-driven site: tagging, content organization, advanced templating, etc. Best of all it's one less application for us to host and keep up-to-date with the latest security patches.
```

Comment:
I'd also make it "Best of all, it's" instead of "Best of all it's" but we know how @AnneTee feels about that.